README.md

```diff
@@ -1,407 +1,436 @@
-# telescope.nvim
-
-Gaze deeply into unknown regions using the power of the moon.
-
-## What is Telescope?
-
-Telescope is a highly extendable fuzzy finder over lists. Items are shown in a popup with a prompt to search over.
-
-Support for:
-
-* LSP (references, document symbols, workspace symbols)
-* Treesitter
-* Grep
-* Files (git, fd, rg)
-* Vim (command history, quickfix, loclist)
-
-[What is Telescope? (Video)](https://www.twitch.tv/teej_dv/clip/RichDistinctPlumberPastaThat)
+# Telescope.nvim
+
+> **Telescope**
+> An arrangement of lenses or mirrors or both that gathers light,
+> permitting direct observation or photographic recording of distant objects.
+>
+> — thefreedictionary
+
+<!-- FIXME: I think `creating floating pickers with advanced features`
+              can be made much clearer for those
+              who don't already know what the plugin is about.
+            Terms like `prompt` and `picker` are unlikely to be familiar. -PL -->
+`Telescope.nvim` is a next generation library for creating floating pickers 
+  with advanced features.
+It is written in `lua` and is built on top of latest
+  awesome features from `neovim` core.
+<!-- FIXME: I don't understand the next sentence. -PL -->
+`Telescope.nvim` is centered around modularity
+  *to the extent that* the promotes can be customized in isolation from one another
+  (such presentation, algorithm, mappings ... etc).
+In addition, `telescope.nvim` comes with a growing number
+  of community driven [built-in pickers](#built-in-pickers),
+  covering a wide range of use cases and tools,
+  and offers a customizable user interface.
+Take a look at our [community gallery](https://github.com/nvim-lua/telescope.nvim/wiki/Gallery)
+  to see some examples.
```

Comment:
We can also mention that people can find extensions in the org.

Reply:
Good Idea, look at the new introduction, it should include it now